backends/cuda-ref/ceed-cuda-ref-vector.c

```diff
@@ -67,14 +67,14 @@ static inline int CeedVectorSyncH2D_Cuda(const CeedVector vec) {
 // Sync device to host
 //------------------------------------------------------------------------------
 static inline int CeedVectorSyncD2H_Cuda(const CeedVector vec) {
-  Ceed             ceed;
   CeedSize         length;
+  Ceed             ceed;
   CeedVector_Cuda *impl;
 
   CeedCallBackend(CeedVectorGetCeed(vec, &ceed));
   CeedCallBackend(CeedVectorGetData(vec, &impl));
 
-  CeedCheck(impl->d_array, ceed, CEED_ERROR_BACKEND, "No valid device data to sync to host");
+  CeedCheck(impl->d_array, CeedVectorReturnCeed(vec), CEED_ERROR_BACKEND, "No valid device data to sync to host");
```

Comment:
```suggestion
  CeedCheck(impl->d_array, ceed, CEED_ERROR_BACKEND, "No valid device data to sync to host");
```

IDC either way, but why change if we already need `ceed` for the rest of the